lighthouserc.yml

```diff
@@ -1,7 +1,30 @@
 ci:
   collect:
     numberOfRuns: 1
+    startServerCommand: sh ./lighthouse.sh
+    startServerReadyPattern: 'BUILD SUCCESSFUL'
+    startServerReadyTimeout: 600000
```

Comment:
The timeout here is set pretty high, but if it's lower there's some flakiness running the tests because lhci doesn't wait long enough for the server to start. It's usually not an issue, but if GitHub resources are in high demand starting the server takes a while and the whole job has to be re-run.